src/cli.ts

```diff
@@ -307,7 +307,7 @@ function logAddressInfo(ctx: Context) {
   log(`secp256k1 public key: 0x${compressedPubKey}`)
 }
 
-async function logBalanceInfo(ctx: Context) {
+export async function logBalanceInfo(ctx: Context) {

```

Comment:
since the function is export please add function description in comment as documentation.